app.py

```diff
@@ -17,21 +17,24 @@ class Users(db.Model):
 	is_admin = db.Column(db.Boolean, nullable=False)
 	is_banned = db.Column(db.Boolean, nullable=False)
 
+
 class Subjects(db.Model):
 	id = db.Column(db.BigInteger, primary_key=True)
 	subject = db.Column(db.Text, unique=True, nullable=False)
 
+
 class Groups(db.Model):
 	id = db.Column(db.BigInteger, primary_key=True)
 	groups = db.Column(db.Text, unique=True, nullable=False)
 
+
 class Posts(db.Model):
 	id = db.Column(db.BigInteger, primary_key=True)
-	user = db.Column(db.BigInteger, db.ForeignKey('users.id'),nullable=False)
-	subject = db.Column(db.BigInteger, db.ForeignKey('subjects.id'),nullable=False)
+	user = db.Column(db.BigInteger, db.ForeignKey('users.id'), nullable=False)
+	subject = db.Column(db.BigInteger, db.ForeignKey('subjects.id'), nullable=False)
 	body = db.Column(db.Text, nullable=False)
 	group = db.Column(db.BigInteger, db.ForeignKey('groups.id'))
-	time = db.Column(db.DateTime, nullable=False, default=datetime.datetime.now)
+	time = db.Column(db.DateTime, nullable=False, default=datetime.now)
```

Comment:
I think you need to send the return of the function and not the function itself => datetime.now() instead datetime.now, check it on google